### PR TITLE
Recommendations for proper versioning of the plugin

### PR DESCRIPTION
**Pull request by advayDev1**
Want to make sure we don't have inconsistencies between the published
versions and the commits in git.

Issue #97 and #101

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,3 +87,18 @@ git push -f
...
+
+We'd like to maintain proper versioning history for the plugin.  This requires
+a little bit of coordination between in-code versions, GitHub tags, and
+published versions on plugins.gradle.org.  The steps are:
+
+1.  As a separate commit, bump the version number in `build.gradle`. Use
+https://semver.org to guide which slot in the version number should be bumped.
+2.  Tag that separate commit as for example `v0.2.3-alpha` and push that
+tag to the j2objc-contrib/j2objc-gradle repository.
+3.  Do a clean build (`./gradlew clean build`), and then publish the new version
```
Comment (brunobowden) [inline CONTRIBUTING.md]: Here's the full command that cleans, builds and publishes:

./gradlew clean publishPlugins

Comment (advayDev1) [inline CONTRIBUTING.md]: Actually it has to be
clean build publishPlugins
because publishPlugins does not run the tests.

Comment (brunobowden) [inline CONTRIBUTING.md]: I went and checked and found the same thing, glad you caught that as well

**Comment by advayDev1** [comment]
@brunobowden ptal

**Comment by brunobowden** [comment]
Looks good

**Pull request merged**